### PR TITLE
Fix crash-stats missing data for yesterday.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Comment by Natim** [comment]
It is only because the data arrive around 10-11am CEST so if you check before you want to take the day before. It is not a loop because it is only yesterday if available or before yesterday.

File: pollbot/tasks/crash_stats.py [hunk 1]
```diff
@@ -37,24 +37,37 @@
...
-                status = Status.ERROR
-                message = "No crash-stats ADI info for version {}".format(versions)
+                # Try the day before
+                date = (datetime.date.today() - datetime.timedelta(days=2)).strftime('%Y-%m-%d')
+
+                url = ('{}/ADI/?start_date={}&end_date={}&'
+                       'platforms=Windows&platforms=Linux&platforms=Mac%20OS%20X&'
+                       'product={}&versions={}')
+                url = url.format(CRASH_STATS_SERVER, date, date, product,
+                                 '&versions='.join(versions))
```
Comment (leplatrem) [inline pollbot/tasks/crash_stats.py]: Isn't there a Python library to generate URLs from tuples?

Comment (leplatrem) [reply pollbot/tasks/crash_stats.py]: Also it really looks like the one above, that's a pity not to share it in a helper or something

File: tests/test_tasks.py [hunk 2]
```diff
@@ -332,10 +332,47 @@ def mock_platforms(self, platforms, working_body):
                'platforms=Mac%20OS%20X&product=firefox&versions=54.0&versions=52.0.2')
         url = url.format(CRASH_STATS_SERVER, date, date)
         self.mocked.get(url, status=200, body='{"hits": [], "total": 0}')
+
+        date = (datetime.date.today() - datetime.timedelta(days=2)).strftime('%Y-%m-%d')
```
Comment (leplatrem) [inline tests/test_tasks.py]: little helper `yesterday()` in utils?

**Comment by Natim** [comment]
> Too late to review apparently...

It is never too late for good comments 👍

**Pull request merged**